### PR TITLE
Vetus Speculator now tell you what type he is

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Now when you examine Vetus, you can see a postscript telling you the type of its core.

## Why It's Good For The Game
Sometimes Vetus' appearance is not obvious, especially to people with color blindness, so being able to understand the core type by inspection can be a lifesaver, as each core type should ideally have different equipment, such as magboots vs cryo, fire protection for fire type, and so on.

## Images of changes
Cryo core example. Now air cold around him 
image

## Testing
do code, try again, do correct

## Changelog
:cl:
tweak: Examination of Vetus now tells us its core type
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm [hunk 1]
```diff
@@ -103,9 +103,21 @@ Difficulty: Hard
...
+		if(GRAV)
+			desc += "There's a distorted gravity around him"
+		if(PYRO)
+			desc += "You see the flames around him"
+		if(FLUX) // Main attack is shock, so flux makes it stronger
+			melee_damage_lower = 25
+			melee_damage_upper = 25
+			desc += "He looks overflowing with energy"
+		if(VORTEX)
+			desc += "You can see the strong wind around him"
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm:118]:
```suggestion
			desc += "It emits sparks of blue energy."
		if(GRAV)
			desc += "There's a distorted gravity around him."
		if(PYRO)
			desc += "You see the flames around him."
		if(FLUX) // Main attack is shock, so flux makes it stronger
			melee_damage_lower = 25
			melee_damage_upper = 25
			desc += "He looks overflowing with energy."
		if(VORTEX)
			desc += "You can see the strong wind around him."
```

**Comment by Scribble-Sheep** [comment]
Vetus gender confirmed?

**Comment by ErdGinalD** [comment]
> Real talk tho, there's a missing space between 'combatant." and "The air" at the end that looks janky in that screencap. I'm not sure how this would normally be handled, but if that's still in the game then there needs to be a space added at the start of those ending messages or something.

Done new version, should be fine like that

File: code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm [hunk 2]
```diff
@@ -103,9 +103,21 @@ Difficulty: Hard
...
+		if(PYRO)
+			desc += "You see flames burning around it."
+		if(FLUX) // Main attack is shock, so flux makes it stronger
+			melee_damage_lower = 25
+			melee_damage_upper = 25
+			desc += "It seems to overflow with energy."
+		if(VORTEX)
+			desc += "You see space bend and distort around it."
+		if(CRYO)
+			desc += "The air surrounding it cold and listless."
```
Comment (Henri215) [inline code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm:120]:
```suggestion
			desc += "The air surrounding it is cold and listless."
```

**Comment by Henri215** [comment]
> > Real talk tho, there's a missing space between 'combatant." and "The air" at the end that looks janky in that screencap. I'm not sure how this would normally be handled, but if that's still in the game then there needs to be a space added at the start of those ending messages or something.
> 
> Done new version, should be fine like that

It still lacks that space he talked about, caused by it not being done over the /examine proc that would add it automatically, since you are doing it on initialize you would have to add a space at the start for all mode descriptions.

**Comment by Scribble-Sheep** [comment]
I can't work out how to do multiline suggestions, sorry

File: code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm [hunk 3]
```diff
@@ -103,9 +103,21 @@ Difficulty: Hard
...
+		if(PYRO)
+			desc += "You see flames burning around it."
+		if(FLUX) // Main attack is shock, so flux makes it stronger
+			melee_damage_lower = 25
+			melee_damage_upper = 25
+			desc += "It seems to overflow with energy."
+		if(VORTEX)
+			desc += "You see space bend and distort around it."
+		if(CRYO)
+			desc += "The air surrounding it is cold and listless."
```
Comment (Scribble-Sheep) [inline code/modules/mob/living/simple_animal/hostile/megafauna/ancient_robot.dm:120]:
```suggestion
			desc += " It emits sparks of blue energy."
		if(GRAV)
			desc += " Gravity seems to distort around it."
		if(PYRO)
			desc += " You see flames burning around it."
		if(FLUX) // Main attack is shock, so flux makes it stronger
			melee_damage_lower = 25
			melee_damage_upper = 25
			desc += " It seems to overflow with energy."
		if(VORTEX)
			desc += " You see space bend and distort around it."
		if(CRYO)
			desc += " The air surrounding it is cold and listless."
```

**Review by Scribble-Sheep — CHANGES_REQUESTED** [review]

**Review by Henri215 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**